modules/aws/vpc/sg-elb.tf

```diff
@@ -68,6 +28,16 @@ resource "aws_security_group_rule" "api_ingress_console" {
   to_port     = 6443
 }
 
+resource "aws_security_group_rule" "tnc_ingress" {
```

Comment:
`tnc` -> `mcs` rebranding?  Or `ignition_ingress`?  Or something.

Reply:
I'd like to keep the rename out of scope here see https://github.com/openshift/installer/pull/242#issuecomment-422946285.

Reply:
You can put it in a different commit, but it belongs in this PR.

Reply:
I think it the rename is not related to the logic of this PR which is to consolidate ELBs and not rename stuff, I added the commit anyway so y'all are happy ;-)